src/containers/ServerStatusControl.tsx

```diff
@@ -0,0 +1,67 @@
+import { IconButton } from '@material-ui/core';
```

Comment:
Renamed the way-old PersistStateControl to ServerStatusControl. This is the white checkmark that appears in the toolbar. Now it shows the status of the websocket connection to the server. If disconnected, a click will attempt reconnect.